src/api/fetch.js

```diff
@@ -1,19 +1,32 @@
 import isomorphicFetch from 'isomorphic-fetch'
 import createQueryString from './createQueryString.js'
-import parseLocation from './parseLocation.js'
 import makeUrl from './makeUrl.js'
 import * as AccessToken from './AccessToken.js'
 import { signinRedirect } from '../utils/redirect.js'
 import log, { error } from '../utils/log.js'
 
+let store = null
```

Comment:
I think a comment here about how this is called from `index.js`, after the store is created there, would be helpful.